site/source/api/templates.md

```diff
@@ -17,8 +17,7 @@ or replaced and should be cleaned up.  You can associate data with a
 template instance, and you can access its DOM nodes when it is in the
 document.
 
-Read more about templates and how to use them in the [Spacebars](#pkg_spacebars)
-package README and the [Blaze](http://guide.meteor.com/blaze.html) article in the Meteor Guide.
+Read more about templates and how to use them in the [Spacebars](http://blazejs.org/api/spacebars) and [Blaze](http://blazejs.org/api/blaze).
```

Comment:
Should we hard-code URLs or should we just have absolute URLs like `/api/spacebars` here? Because it is easier then to work locally on this content.

Not sure if Hexo has some way to specify URL placeholders which it then resolves as it is compiling markdown?